publications-2011.bib

```diff
@@ -197,12 +198,13 @@ @phdthesis{2011_18
 @article{2011_19,
    author  = {T. George and A. Gupta and V. Sarin},
    title   = {An empirical analysis of the performance of iterative solvers for SPD systems},
-   journal = {Submitted to ACM Trans. Math. Softw., 2011. IBM Research Report RC 24737},
+   journal = {ACM Trans. Math. Softw., 2011. IBM Research Report RC 24737},
```

Comment:
same here and it is misleading (I assume this never got accepted)

Reply:
So how should we proceed here? Would you like me to clean up the entry now or should we do this in another PR?

Reply:
I am fine with leaving them as is. I mean, if we have a "note = ..." then it means we need to look at them eventually. And by "we" I don't mean you.